static/js/templates.js

```diff
@@ -2,6 +2,14 @@ var templates = (function () {
 
 var exports = {};
 
+function trans(str, context) {
+    if (typeof i18n !== 'undefined') {
+        // only translate the string if i18n is loaded
+        return i18n.t(str, context);
+    }
+    return str;
+}
```

Comment:
I think the problem can only happen in templates which can be accessed directly through a url e.g. Settings page and Administration page.